common/src/main/java/com/twilio/video/examples/common/VideoFrameKtx.kt

```diff
@@ -19,22 +19,20 @@ import tvi.webrtc.YuvConverter
  * valid EGL context when the frame buffer is a [VideoFrame.TextureBuffer].
  */
 fun VideoFrame.toBitmap(): Bitmap? {
-    val i420Buffer = if (buffer is VideoFrame.TextureBuffer) {
+    // Construct yuv image from image data
+    val yuvImage = if (buffer is VideoFrame.TextureBuffer) {
         val yuvConverter = YuvConverter()
-        val buffer = yuvConverter.convert(buffer as VideoFrame.TextureBuffer)
+        val i420Buffer = yuvConverter.convert(buffer as VideoFrame.TextureBuffer)
         yuvConverter.release()
-        buffer
+        i420ToYuvImage(i420Buffer, i420Buffer.width, i420Buffer.height)
     } else {
-        buffer.toI420()
+        val i420Buffer = buffer.toI420()
+        val returnImage = i420ToYuvImage(i420Buffer, i420Buffer.width, i420Buffer.height)
+        i420Buffer.release()
```

Comment:
What happen if the `i420Buffer` is released before the Image creation? (Before line 30)
is there any difference releasing it before or after?

Reply:
Ok got, but I was thinking... in the `else` block, is there any difference between "releasing" the buffer in the line 31 and before line 30?

If nothing happen, would it be better to create and return the image in line 32.